master/internal/user/postgres_users_intg_test.go

```diff
@@ -0,0 +1,151 @@
+//go:build integration
+// +build integration
+
+package user
+
+import (
+	stdContext "context"
+	"fmt"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/determined-ai/determined/master/internal/db"
+	"github.com/determined-ai/determined/master/pkg/etc"
+
+	"github.com/determined-ai/determined/master/pkg/model"
+)
+
+func setupBun(t *testing.T) {
+	pgDB = db.MustResolveTestPostgres(t)
+	db.MustMigrateTestPostgres(t, pgDB, "file://../../static/migrations")
+	require.NoError(t, etc.SetRootPath("../../static/srv"))
+}
+
+func TestPostgresUsers(t *testing.T) {
```

Comment:
this is much more compact than the tests I wrote for templates. I weighed this vs what I did and went with what I did because the test failures much more clearly identify what feature in template failed and the tests are easier to maintain when refactoring since a change to one API affects one small test, not a lot to do with one large test.

wdyt?

Reply:
see new commit!